lib/user.rb

```diff
@@ -16,5 +18,18 @@ def initialize(input)
     def add_trip(trip)
       @trips << trip
     end
+
+    def net_expenditures
+      costs = @trips.map {|trip_stuff| trip_stuff.cost}
+        cost_total = costs.inject(:+).to_f
+        return cost_total
```

Comment:
Two comments here. First, `trip_stuff` is not a great variable name, something like `trip` or `single_trip` might be better.

Second, instead of `.inject(:+)` you could say `.sum`, which I think does a better job of clarifying your intent here.